xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -53,7 +53,7 @@
 
 ## Remarks
 
-The memory for the consumed data will be released and no longer available. The examined data communicates to the pipeline when it should signal more data is available.
+The memory for the consumed data will be released and no longer available. The `ReadResult.Buffer` previously returned from `ReadAsync(CancellationToken)` must not be accessed after this call. The examined data communicates to the pipeline when it should signal more data is available.
```

Comment:
These could be xref links:

```suggestion
The memory for the consumed data will be released and no longer available. The <xref:System.IO.Pipelines.ReadResult.Buffer%2A> previously returned from <xref:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)> must not be accessed after this call. The examined data communicates to the pipeline when it should signal more data is available.
```

Reply:
I like that. I didn't know how to do it in markdown, but this is the same kind of thing I did in the original xml doc comments.